samples/Makefile

```diff
@@ -11,40 +11,74 @@ generate:
 	@echo  $(TEST_PKGS) $(TEST_DB_PKG)
 	
 	../bin/wag -with-tests -file ./swagger.yml -output-path ./gen-go-basic -js-path ./gen-js
-	cd ./gen-js && jsdoc2md index.js types.js > ./README.md
+	cd ./gen-js && jsdoc2md index.js types.js > ./README.md    
+	cd ./gen-go-basic/client && go mod tidy
+	cd ./gen-go-basic/models && go mod tidy
```

Comment:
Can you use a function here. For most of the cases it looks like this. 
```
define gen-tidy
@cd $(1) && jsdoc2md index.js types.js > ./README.md
@cd $(2)/client && go mod tidy
@cd $(2)/models && go mod tidy
enddef
```

Reply:
Most of them are, so I suppose I could use a function for most.

Reply:
Hmm, good idea, if I'm going that route might as well clean this up even more.